src/pages/Staking/index.tsx

```diff
@@ -692,7 +701,9 @@ const Staking = (): JSX.Element => {
 
   const lockTimeFieldDisabled =
     votingBalanceGreaterThanZero === undefined ||
-    remainingBlockNumbersToUnstake === undefined;
+    remainingBlockNumbersToUnstake === undefined ||
+    availableLockTime === undefined ||
+    availableLockTime <= 0;
```

Comment:
The point is this change. I made it so that the lock time field is disabled when `availableLockTime` is less than zero.